code/src/components/PreferenceQuestion.js

```diff
@@ -0,0 +1,45 @@
+/* eslint-disable jsx-a11y/no-noninteractive-tabindex */
+/* eslint-disable jsx-a11y/label-has-associated-control */
+import React from 'react'
+// import Alert from './Alert';
+
+// eslint-disable-next-line max-len
+const PreferenceQuestion = ({ preference, onPreferenceChange, onStepChange, onEnter, onPreviousQuestionChange }) => {
+  return (
+    <div className="survey-container">
+      <form className="form-question">
+        <div tabIndex="0" className="label-text">How is the weather? Are you in the mood for some hot or cold food?</div>
+        <div className="preference-container">
+          <label className="label-input-preference" htmlFor="preference">
+            <input
+              id={preference}
+              type="radio"
+              value="hot"
+              onChange={onPreferenceChange}
+              checked={preference === 'hot'}
+              onKeyPress={onEnter}
+              alt="hot" />
+            <p>Hot food</p>
+          </label>
+          <label className="label-input-preference" htmlFor="preference">
+            <input
+              id={preference}
+              type="radio"
+              value="cold"
+              onChange={onPreferenceChange}
+              checked={preference === 'cold'}
+              onKeyPress={onEnter}
```

Comment:
On your forms you could have called the onStepChange instead of making a new function called onEnter since they are doing the same thing. So it would look like this:
```onKeyPress={onStepChange}```

Reply:
Ooh, I have to try it out! Thank you for all the good feedback!